src/common/utils/formatDateForChat.ts

```diff
@@ -0,0 +1,30 @@
+import { format } from 'date-fns';
+import { t } from 'i18next';
+
+function formatDateForChatList(date: Date): string {
```

Comment:
I dont see test for this function, all utils should be covered

Reply:
Added tests for this function